cron/poll_read_issue_comments.py

```diff
@@ -116,8 +117,12 @@ def insert_or_update(api, comment_id, issue_id, comment_txt):
 
     voting_window = gh.voting.get_initial_voting_window()
 
-    seconds_remaining = gh.issues.voting_window_remaining_seconds(api, settings.URN, comment_id,
-                                                                  voting_window)
+    try:
+        seconds_remaining = gh.issues.voting_window_remaining_seconds(api, settings.URN, comment_id,
+                                                                      voting_window)
+    except HTTPError:
+        # Possible comment deleted
+        pass  # handle later
     seconds_remaining = max(0, seconds_remaining)  # No negative time
```

Comment:
If an exception happens above, it will crash here. Also handle the exception with `__log.exception` please.